installer/bundle/bundle_skel_Linux.sh

```diff
@@ -620,6 +621,10 @@ case "$installMode" in
             if [ $? -eq 0 ]; then
                 shouldInstall_omi
                 pkg_upd_list $OMI_PKG omi $?
+            else
+                pkg_add_list $OMI_PKG omi
+                pkg_add ${ADD_PKG_QUEUE}
+                OMI_EXIT_STATUS=$?
```

Comment:
If you're doing an update operation with the shell bundle, then you should UPDATE omi (even if it's not installed), and that **must** be done during the same operation as the upgrade for SCX.

In the end, the RPM command line should be something like:

```
rpm -Uvh <omi-rpm-file> <scx-rpm-file>
```

Combining these updates into a single operation was the entire point of this change.

Please let me know if you have further questions.